cfme/tests/containers/test_basic_metrics.py

```diff
@@ -18,7 +21,7 @@ def test_basic_metrics(provider):
     routes = provider.mgmt.o_api.get('route')[1]['items']
     metrics_hostname = [route for route in routes
                         if route['metadata']['name'] == 'hawkular-metrics']
-    assert metrics_hostname, 'Could not find route for hawkular-metrics'
+    assert metrics_hostname, 'Could not find route for metrics'
     metrics_hostname = metrics_hostname.pop()['spec']['host']
     host_url = 'https://' + metrics_hostname + '/hawkular/metrics/'
```

Comment:
Validating Prometheus endpoint is done by sending a request to ```https://<Promethues metrics route/graph``` and it requires add of ```Authorization``` header to the request e.g: ```curl -H "Authorization: Bearer <OCP Token>" -k https:/<Prometheus metrics route>/graph```

Reply:
In addition, I think using ```curl``` command here is less appropriate, I would prefer using  python's ```requests``` module